docs/developers/coredev/core_dev_guide.md

```diff
@@ -95,9 +95,13 @@ version numbers `0.x` and do not have a deprecation policy, but we will work to
 4. **Documentation and tutorials:** All new methods should have appropriate doc
 strings following [PEP257](https://peps.python.org/pep-0257/) and the
 [NumPy documentation guide](https://numpy.org/devdocs/dev/howto-docs.html#documentation-style).
-For any major new features, accompanying changes should be made to our
-[tutorials](tutorials). These should not only
-illustrates the new feature, but explains it. Titles for all documents in napari should follow [sentence case capitalization](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case), but the name `napari` should always be written in lowercase.
+For API changes, make sure to check that a `.. versionadded::` or
+`.. versionchanged::` directive has been added to the appropriate docstring. For
+any major new features, accompanying changes should be made to our
+[tutorials](tutorials). These should not only illustrate the new feature, but
+explain it. Titles for all documents in napari should follow
+[sentence case capitalization](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case),
+but the name `napari` should always be written in lowercase.
```

Comment:
Glad to have this documented ! The other question, should napari always be in backticks?

Reply:
I wonder if this part should be added somewhere more generally for all users (not just core devs) e.g., in our guide to contributing documentation?

Reply:
No I think this was meant to show how you should write napari in the sources.

I thought about it and decided against adding this to the general "contributing docs" pages for two reasons. One is that if you are contributing to the docs in general you are not going to be changing APIs so you will not face this case. The other is that I see adding this directive in general as the job of the core team, in that you should probably suggest this on review same as you would do for adding a release note, for example. So as long as the core team is informed, I think we are good.

Reply:
So do you have a suggestion on the wording to use instead of the note I added above to the contributing/index doc?